fpdcc/ccfp-asset-dashboard
# Project detail improvements

## Overview

Closes:
- #158 
- #159 
- #160
- Includes some UI rearrangement/style changes. Not the best UI but it's a WIP.

## Testing Instructions
- visit https://ccfp-asset-d-feature-pr-w4uita.herokuapp.com
- credentials: `ccfp-asset-dashboard test user`
- add a project with new project manager attribute
- edit the project with new countywide attribute
- add a phase to the project
- make sure the phase shows up in the table on the project overview page
